databrowse/support/cef_web_support.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python
```

Comment:
Is this file significantly different from db_web_support?  If not, then we should also keep those together or we should figure out how to refactor all of the "web_support" files to reduce code duplication.  It's not fun maintaining three copies of the same code.